lib/src/base-styles/_nx-lists.scss

```diff
@@ -128,109 +120,65 @@
  * Action or Clickable Lists
  */
 .nx-list--clickable {
-  .nx-list__title {
-    color: $nx-text-color;
-  }
-
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
-      pointer-events: none;
-    }
+      background-color: #d3e6ff;
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
-      &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
-        cursor: pointer;
-        outline-style: none; // removes :focus outline
-
-        .fa-chevron-right {
-          color: $nx-color-blue-41;
-        }
-      }
-
-      &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
+      &:hover {
```

Comment:
The line above could be removed if this was `&, &:hover`

Reply:
> The line above could be removed if this was `&, &:hover`

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/f2e21198ea5496332e690d414f65c30e7f6c8bdb